src/desktop/version.cpp

```diff
@@ -183,22 +152,6 @@ std::string os_version()
 			return ver;
 		}
 	}
-#endif
-
-	//
-	// POSIX uname version.
```

Comment:
Please don't remove this code. It's a fallback in case `/etc/lsb_release` doesn't exist.

Reply:
Done - 1bb926a